.gitignore

```diff
@@ -1,3 +1,5 @@
+Gemfile.lock
+.idea/
```

Comment:
If this is rubymine specific do you mind moving it to your [global gitnore](https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore)?